src/utils/__tests__/__snapshots__/getConfig.test.js.snap

```diff
@@ -34,7 +34,7 @@ exports[`creates config: diff ios/android config 1`] = `
     \\"output\\": Object {
 -     \\"filename\\": \\"index.ios.bundle\\",
 +     \\"filename\\": \\"index.android.bundle\\",
-      \\"path\\": \\"<<REPLACED>>/src/utils/__tests__/fixtures\\",
+      \\"path\\": undefined,
```

Comment:
is this expected?

Reply:
Yes. This comes from https://github.com/callstack/haul/pull/516/files#diff-e573bff19b6673d9da63d58f54edd482R24 and is the reason this fix works.

Reply:
I actually don't want to pass this in tests, since they generate configs in a way I didn't anticipate.